txnprovider/txpool/pool_txn_parser.go

```diff
@@ -171,7 +171,7 @@ func (ctx *TxnParseContext) ParseTransaction(payload []byte, pos int, slot *TxnS
 		}
 		// For legacy transaction, the entire payload in expected to be in "rlp" field
 		// whereas for non-legacy, only the content of the envelope (start with position p)
-		slot.Rlp = payload[p-1 : dataPos+dataLen]
+		slot.Rlp = payload[0 : dataPos+dataLen]
```

Comment:
@shohamc1 
There seemed to be an issue with `p = 2` for AA transactions causing strange error messages in the logs. This looks a bit dangerous and also, we have removed the "subtype" byte from the RIP-7560 lately, so this entire situation can be avoided.